src/client/src/ui/spotTile/views/tradeNotification.jsx

```diff
@@ -30,7 +30,7 @@ export default class TradeNotification extends React.Component {
         'trade-notification',
         className,
         {
-          'trade-notification--rejected': tradeExecutionNotification.status === TradeStatus.Rejected
+          'trade-notification--rejected': tradeExecutionNotification.status === TradeStatus.Rejected.name
```

Comment:
Actually shouldn't `tradeExecutionNotification.status` be a `TradeStatus`, if so you'd not need the `.name`